runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerOverlay.java

```diff
@@ -65,7 +64,7 @@
 		ItemID.SLAYER_RING_8
 	);
 
-	private final Set<Integer> slayerEquipment = ImmutableSet.of(
+	private final static ImmutableIntArray ALL_SLAYER_ITEMS = ImmutableIntArray.of(
```

Comment:
as it is static now it should be at the top of the class. Also, while at it, slayerJewelry should receive same treatment

Reply:
Roger that